templates/zabbix-agent-systemd.init.erb

```diff
@@ -1,16 +1,16 @@
 [Unit]
 Description=Zabbix Agent
-After=syslog.target network.target
-Documentation=man:zabbix_agentd
+After=syslog.target
+After=network.target
 
 [Service]
 Type=forking
-ExecStart=/usr/sbin/zabbix_agentd -c <%= @agent_configfile_path %>
-Restart=on-abnormal
+Restart=on-failure
 PIDFile=<%= @pidfile %>
-PrivateTmp=yes
-ProtectSystem=full
-ProtectHome=yes
```

Comment:
so these additional security settings are pretty smart, but not upstream, I would prefer if we keep them or at least open up a ticket upstream to mention these.